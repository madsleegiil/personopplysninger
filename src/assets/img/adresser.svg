<?xml version="1.0" encoding="UTF-8"?>
<svg width="62px" height="62px" viewBox="0 0 62 62" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
    <title>book-contact-2 Copy 2</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="book-contact-2-Copy-2" fill-rule="nonzero">
            <path d="M48.8972041,17.4675245 C49.1191722,17.6871982 49.4151295,17.8051282 49.7157112,17.8051282 C50.3515573,17.8051282 50.8717949,17.2871608 50.8717949,16.6489511 L50.8717949,6.24335665 C50.8717949,5.60514685 50.3538694,5.08717949 49.7157112,5.08717949 L39.310959,5.08717949 C38.8439012,5.08717949 38.4207748,5.36928672 38.2427378,5.80169698 C38.0623888,6.23179487 38.161812,6.72895104 38.4924518,7.05961771 L48.8972041,17.4675245 Z" id="Fill/home-1" fill="#78706A"></path>
            <polygon id="Path" fill="#C86151" points="5.08717949 23.5898947 5.08717949 61.0461538 22.8923077 61.0461538 22.8923077 38.1538462 38.1538462 38.1538462 38.1538462 61.0461538 55.9589744 61.0461538 55.9589744 23.5898947 30.5230769 -5.32907052e-14"></polygon>
            <path d="M60.6966193,26.1090918 C60.6864456,26.1023742 60.6813587,26.0844605 60.6711852,26.0732646 L31.4219038,0.322445044 C30.9361114,-0.107481681 30.1095013,-0.107481681 29.6237089,0.322445044 L0.374427556,26.0732646 C-0.0859306977,26.47856 -0.152059508,27.1458422 0.346449983,27.620553 C0.923805362,28.1288517 1.69954716,28.054958 2.15481858,27.6630978 C2.15990543,27.6608585 2.16753567,27.6586194 2.1726225,27.6563802 L30.524078,2.69823804 L58.8729903,27.6586194 C58.878077,27.6608585 58.8831638,27.6630978 58.8882507,27.665337 C59.3714997,28.0795892 60.1726756,28.0795892 60.6533811,27.665337 C60.658468,27.6630978 60.6660983,27.6608585 60.6711852,27.6586194 C61.1340869,27.2510846 61.1976721,26.5748457 60.6966193,26.1090918 Z" id="Path" fill="#59514B"></path>
        </g>
    </g>
</svg>