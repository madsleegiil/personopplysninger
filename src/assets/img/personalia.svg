<?xml version="1.0" encoding="UTF-8"?>
<svg width="52px" height="62px" viewBox="0 0 52 62" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
    <title>book-contact-2 Copy</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="book-contact-2-Copy">
            <path d="M6.74545455,0 L49.1454545,0 L49.1454545,12.7179487 L6.74545455,12.7179487 C3.19744342,12.7179487 0.321212121,9.87093892 0.321212121,6.35897436 C0.321212121,2.84700979 3.19744342,6.45137425e-16 6.74545455,0 Z" id="Rectangle" fill="#E7E9E9"></path>
            <path d="M50.4303032,10.1743589 L6.74545447,10.1743589 C4.62031506,10.1743589 2.89090899,8.46252298 2.89090899,6.35897426 C2.89090899,4.25542553 4.62031506,2.54358963 6.74545447,2.54358963 L50.4303032,2.54358963 C51.1395396,2.54358963 51.7151517,1.97382553 51.7151517,1.27179475 C51.7151517,0.572307567 51.1395396,-1.27999027e-07 50.4303032,-1.27999027e-07 L6.74545447,-1.27999027e-07 C3.20184233,-1.27999027e-07 0.321212009,2.85390758 0.321212009,6.35897426 C0.321212009,6.65911785 0.341769585,6.95417426 0.380315039,7.2415999 C0.341769585,7.3662358 0.321212009,7.49595888 0.321212009,7.63076913 L0.321212009,54.6871796 C0.321212009,58.1922463 3.20184233,61.046154 6.74545447,61.046154 L50.4303032,61.046154 C51.1395396,61.046154 51.7151517,60.4763899 51.7151517,59.7743591 L51.7151517,11.4461538 C51.7151517,10.7466666 51.1395396,10.1743589 50.4303032,10.1743589 Z" id="Fill/book-contact-2" fill="#D6897D" fill-rule="nonzero"></path>
            <path d="M37.5818183,48.3282051 L14.4545455,48.3282051 C13.7453091,48.3282051 13.169697,47.758441 13.169697,47.0564103 C13.169697,39.3010051 17.2966303,35.5542974 21.2976485,34.0001641 C19.4834424,32.6037333 18.3090909,30.4264205 18.3090909,27.9794872 C18.3090909,23.7723897 21.767903,20.3487179 26.0181818,20.3487179 C30.2684606,20.3487179 33.7272729,23.7723897 33.7272729,27.9794872 C33.7272729,30.4264205 32.5529214,32.6037333 30.7387152,34.0001641 C34.7397335,35.5517538 38.8666668,39.3010051 38.8666668,47.0564103 C38.8666668,47.758441 38.2910547,48.3282051 37.5818183,48.3282051 Z" id="Path" fill="#E7E9E9" fill-rule="nonzero"></path>
            <path d="M6.74545447,5.08717938 C6.0362181,5.08717938 5.46060598,5.65948707 5.46060598,6.35897426 C5.46060598,7.06100503 6.0362181,7.63076914 6.74545447,7.63076914 L47.8606062,7.63076914 C48.5698426,7.63076914 49.1454547,7.06100503 49.1454547,6.35897426 C49.1454547,5.65948707 48.5698426,5.08717938 47.8606062,5.08717938 L6.74545447,5.08717938 Z" id="Path" fill="#C6C2BF"></path>
        </g>
    </g>
</svg>