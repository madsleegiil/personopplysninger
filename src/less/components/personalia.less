.personalia-list {
  list-style: none;
  columns: 2;

  li {
    padding: 1rem;

    .title {
      font-weight: bold;
      display: block;
      white-space: nowrap;
      text-overflow: ellipsis;
      overflow: hidden;
    }
  }
}
