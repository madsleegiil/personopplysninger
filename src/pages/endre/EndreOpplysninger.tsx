import React from "react";
import Adresser from "pages/forside/sections/4-personinfo/3-adresser/Adresser-PDL";
import Utbetalinger from "../forside/sections/4-personinfo/4-utbetalinger/Utbetalinger";
import Box from "components/box/Box";
import kontaktIkon from "assets/img/Kontakt.svg";
import RedirectKnapp from "components/knapper/Redirect";
import { withRouter, RouteComponentProps } from "react-router";
import PDLTelefonnummerHosNav from "../forside/sections/4-personinfo/2-kontaktinfo/subsections/TelefonnummerHosNav-PDL";
import MedPersonInfo from "../../providers/personinfo/PersinInfo";

interface Routes {
  tjeneste?: string;
}

const EndreAlleOpplysninger = (props: RouteComponentProps<Routes>) => {
  const { tjeneste } = props.match.params;
  return (
    <div className="endreOpplysninger__page">
      <div className="endreOpplysninger__container pagecontent">
        {tjeneste && <RedirectKnapp tjeneste={tjeneste} />}
        <MedPersonInfo>
          {({ personalia, adresser }) => {
            return (
              <>
                {personalia && (
                  <Box
                    id="kontaktinformasjon"
                    tittel="kontaktinfo.tittel"
                    icon={kontaktIkon}
                  >
                    <PDLTelefonnummerHosNav tlfnr={personalia.tlfnr} />
                  </Box>
                )}
                {adresser && <Adresser adresser={adresser} />}
                {personalia && (
                  <Utbetalinger
                    kontonr={personalia.kontonr}
                    utenlandskbank={personalia.utenlandskbank}
                  />
                )}
              </>
            );
          }}
        </MedPersonInfo>
      </div>
    </div>
  );
};

export default withRouter(EndreAlleOpplysninger);
