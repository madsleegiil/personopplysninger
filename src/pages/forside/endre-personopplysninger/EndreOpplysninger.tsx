import React from "react";
import Adresser from "pages/forside/sections/4-personinfo/3-adresser/Adresser-PDL";
import Utbetalinger from "pages/forside/sections/4-personinfo/4-utbetalinger/Utbetalinger";
import Box from "components/box/Box";
import { useParams } from "react-router-dom";
import kontaktIkon from "assets/img/Kontakt.svg";
import RedirectKnapp from "components/knapper/Redirect";
import PDLTelefonnummerHosNav from "pages/forside/sections/4-personinfo/2-kontaktinfo/subsections/TelefonnummerHosNav-PDL";
import MedPersonInfo from "providers/personinfo/PersinInfo";
import Spinner from "components/spinner/Spinner";

interface Routes {
  tjeneste?: string;
  redirectUrl?: string;
}

const EndreAlleOpplysninger = () => {
  const params = useParams<Routes>();
  const { tjeneste, redirectUrl } = params;

  return (
    <div className="endreOpplysninger__page">
      <div className="endreOpplysninger__container pagecontent">
        {tjeneste && redirectUrl && (
          <RedirectKnapp tjeneste={tjeneste} redirectUrl={redirectUrl} />
        )}
        <MedPersonInfo loader={<Spinner />}>
          {({ personalia, adresser }) => {
            return (
              <>
                {personalia && (
                  <Box
                    id="kontaktinformasjon"
                    tittel="kontaktinfo.tittel"
                    icon={kontaktIkon}
                  >
                    <PDLTelefonnummerHosNav tlfnr={personalia.tlfnr} />
                  </Box>
                )}
                {adresser && <Adresser adresser={adresser} />}
                {personalia && (
                  <Utbetalinger
                    kontonr={personalia.kontonr}
                    utenlandskbank={personalia.utenlandskbank}
                  />
                )}
              </>
            );
          }}
        </MedPersonInfo>
      </div>
    </div>
  );
};

export default EndreAlleOpplysninger;
