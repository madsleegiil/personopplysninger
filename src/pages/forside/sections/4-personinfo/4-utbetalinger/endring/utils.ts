import { OptionType } from "types/option";
import { Fields } from "calidation";
import {
  LAND_MED_BANKKODE,
  BIC,
} from "./utenlandsk-bankkonto/UtenlandsBankkonto";

import { LandOppslag } from "./landOppslag";

export const getCountryAlpha2 = (countryCode: string) =>
  LandOppslag.has(countryCode) ? LandOppslag.get(countryCode)?.alpha2 : "";

export const getIbanPrefixAlternatives = (countryCode: string): string[] => {
  return LandOppslag.has(countryCode)
    ? LandOppslag.get(countryCode)?.ibanPrefixAlternatives || []
    : [];
};

export const harValgtBic = (bankidentifier?: string) =>
  !!(bankidentifier && bankidentifier === BIC);

export const harValgtUSA = (land?: OptionType) =>
  !!(land && land.value === "USA");

export const brukerBankkode = (land?: OptionType) =>
  !!(land && LAND_MED_BANKKODE.includes(land.value));

export const validerBic = (fields: Fields) => {
  if (harValgtUSA(fields.land)) {
    return false;
  }

  if (brukerBankkode(fields.land)) {
    if (harUtfylt(fields.bickode) || !harUtfylt(fields.bankkode)) {
      return true;
    }
  }

  if (!brukerBankkode(fields.land)) {
    return true;
  }

  return false;
};

export const validerBankkode = (fields: Fields) => {
  if (harValgtUSA(fields.land)) {
    return true;
  }

  if (brukerBankkode(fields.land)) {
    if (harUtfylt(fields.bankkode) || !harUtfylt(fields.bickode)) {
      return true;
    }
  }

  return false;
};

export const harUtfylt = (value?: string) => (value ? true : false);

export const erLandIEuropa = (land: OptionType): boolean => {
  if (!land) {
    return false;
  }
  return !!(
    LandOppslag.has(land.value) && LandOppslag.get(land.value)?.isEurope
  );
};
