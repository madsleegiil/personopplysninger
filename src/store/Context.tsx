import React, { createContext, useContext, useReducer, Dispatch } from "react";
import { Action, Locale, reducer, Store } from "./Store";
import { FetchNameInfo } from "./providers/Auth";
import { FetchFeatureToggles } from "./providers/FeatureToggles";
import { FetchDsopInfo } from "../pages/digital-samhandling-offentlig-privat/DsopFetch";
import { FetchInstInfo } from "../pages/institusjonsopphold/InstFetch";
import { FetchPersonInfo } from "./providers/PersonInfo";
import { FetchKontaktInfo } from "../pages/forside/sections/4-personinfo/2-kontaktinfo/subsections/kontakt-og-reservasjonsregisteret/DKIF-Fetch";
import { Fetchskattetrekksmeldinger } from "../pages/skattetrekksmelding/SkattFetch";
import { FetchMedlInfo } from "../pages/medlemskap-i-folketrygden/MedlFetch";

interface Props {
  children: JSX.Element | JSX.Element[];
}

export const StoreContext = createContext({} as [Store, Dispatch<Action>]);
export const StoreProvider = (props: Props) => {
  const { children } = props;

  const initialLocale = (window.location.pathname.includes("/en")
    ? "en"
    : "nb") as Locale;

  const initialState = {
    locale: initialLocale,
    nameInfo: { status: "LOADING" } as FetchNameInfo,
    featureToggles: {
      status: "LOADING",
      data: {
        "personopplysninger.pdl": false,
        "personopplysninger.dsop": false,
        "personopplysninger.inst": false,
        "personopplysninger.skatt": false,
        "personopplysninger.medl": false,
        "personopplysninger.fullmakt": false,
        "personopplysninger.tilrettelegging": false,
        "pdl-fullmakt": false,
      },
    } as FetchFeatureToggles,
    dsopInfo: { status: "LOADING" } as FetchDsopInfo,
    instInfo: { status: "LOADING" } as FetchInstInfo,
    personInfo: { status: "LOADING" } as FetchPersonInfo,
    kontaktInfo: { status: "LOADING" } as FetchKontaktInfo,
    skattetrekksmeldinger: { status: "LOADING" } as Fetchskattetrekksmeldinger,
    medlInfo: { status: "LOADING" } as FetchMedlInfo,
  };

  return (
    <StoreContext.Provider value={useReducer(reducer, initialState)}>
      {children}
    </StoreContext.Provider>
  );
};
export const useStore = () => useContext(StoreContext);
