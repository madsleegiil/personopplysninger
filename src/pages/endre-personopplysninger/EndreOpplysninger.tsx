import React from "react";
import AdresserVisning from "pages/forside/sections/4-personinfo/3-adresser/Adresser";
import Utbetalinger from "pages/forside/sections/4-personinfo/4-utbetalinger/Utbetalinger";
import Box from "components/box/Box";
import kontaktIkon from "assets/img/Kontakt.svg";
import TelefonnummerHosNav from "pages/forside/sections/4-personinfo/2-kontaktinfo/subsections/TelefonnummerHosNav";
import { Personalia } from "../../types/personalia";
import { Adresser } from "../../types/adresser";

interface Props {
  personalia?: Personalia;
  adresser?: Adresser;
}

const EndreOpplysninger = (props: Props) => {
  const { personalia, adresser } = props;
  return (
    <>
      {personalia && (
        <Box
          id="kontaktinformasjon"
          tittel="kontaktinfo.tittel"
          icon={kontaktIkon}
        >
          <TelefonnummerHosNav tlfnr={personalia.tlfnr} />
        </Box>
      )}
      {adresser && <AdresserVisning adresser={adresser} />}
      {personalia && (
        <Utbetalinger
          kontonr={personalia.kontonr}
          utenlandskbank={personalia.utenlandskbank}
        />
      )}
    </>
  );
};

export default EndreOpplysninger;
