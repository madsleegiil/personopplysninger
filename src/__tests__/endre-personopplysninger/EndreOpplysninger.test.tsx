import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import personInfo from "../../clients/apiMock/app/fetch/person-info.json";
import { IntlProvider } from "react-intl";
import nbMessages from "text/nb";
import EndreOpplysninger from "../../pages/endre-personopplysninger/EndreOpplysninger";
import { Adresser } from "../../types/adresser";
import { Personalia } from "../../types/personalia";
import { StoreProvider } from "store/Context";
import { ValidatorsProvider } from "calidation";
import { extraValidators } from "../../utils/validators";

jest.mock("react-modal");
jest.mock("nav-frontend-js-utils", () => ({
  ...jest.requireActual("nav-frontend-js-utils"),
  guid: () => "Mocked UUID",
}));

describe("EndreOpplysninger", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ValidatorsProvider validators={extraValidators}>
          <StoreProvider>
            <IntlProvider locale={"nb"} messages={nbMessages}>
              <EndreOpplysninger
                adresser={personInfo.adresser as unknown as Adresser}
                personalia={personInfo.personalia as unknown as Personalia}
              />
            </IntlProvider>
          </StoreProvider>
        </ValidatorsProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
