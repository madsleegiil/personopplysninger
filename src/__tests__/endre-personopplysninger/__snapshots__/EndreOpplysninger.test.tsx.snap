// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EndreOpplysninger renders correctly 1`] = `
Array [
  <div
    className="box__wrapper"
    id="kontaktinformasjon"
  >
    <div
      className="nav-veilederpanel nav-veilederpanel--kompakt nav-veilederpanel--plakat"
    >
      <div
        className="nav-veileder nav-veileder--flytende"
      >
        <div
          className="nav-veileder__frame nav-veileder__frame--m"
        >
          <img
            alt="Veileder"
            className="box__ikon"
            src="Kontakt.svg"
          />
        </div>
      </div>
      <div
        className="nav-veilederpanel__content"
      >
        <div
          className="box__container"
        >
          <div
            className="box__header"
          >
            <div
              className="box__title-container"
            >
              <div
                className="box__line"
              />
              <h2
                className="typo-systemtittel box__title"
              >
                Kontaktinformasjon
              </h2>
              <div
                className="box__line"
              />
            </div>
          </div>
          <div
            className="box__content"
          >
            
            <div
              className="underseksjon__header"
            >
              <h2
                className="typo-undertittel"
              >
                Telefonnummer registrert hos NAV
              </h2>
            </div>
            <div>
              <form
                className="tlfnummer__rad"
                noValidate={true}
                onChange={[Function]}
                onReset={[Function]}
                onSubmit={[Function]}
                validators={
                  Object {
                    "hasMultipleCombinedSpaces": [Function],
                    "isBIC": [Function],
                    "isBICCountryCompliant": [Function],
                    "isBankkode": [Function],
                    "isBlacklisted": [Function],
                    "isBlacklistedCommon": [Function],
                    "isEmail": [Function],
                    "isEqual": [Function],
                    "isExactLength": [Function],
                    "isFirstCharNotSpace": [Function],
                    "isGreaterThan": [Function],
                    "isHouseNumber": [Function],
                    "isIBAN": [Function],
                    "isIBANCountryCompliant": [Function],
                    "isIBANRequired": [Function],
                    "isLessThan": [Function],
                    "isLetters": [Function],
                    "isLettersAndDigits": [Function],
                    "isLettersAndSpace": [Function],
                    "isLettersSpaceAndDigits": [Function],
                    "isMaxLength": [Function],
                    "isMinLength": [Function],
                    "isMinOneLetter": [Function],
                    "isMod11": [Function],
                    "isNormalizedLength": [Function],
                    "isNotIBAN": [Function],
                    "isNotSSN": [Function],
                    "isNotYourSSN": [Function],
                    "isNumber": [Function],
                    "isPositive": [Function],
                    "isRegexMatch": [Function],
                    "isRequired": [Function],
                    "isValidAdresselinje": [Function],
                    "isValidBanknavn": [Function],
                    "isValidNorwegianNumber": [Function],
                    "isValidStreetName": [Function],
                    "isWhitelisted": [Function],
                    "notOnlySignsDigitsSpace": [Function],
                    "notOnlySignsSpace": [Function],
                  }
                }
              >
                <div
                  className="tlfnummer__container"
                >
                  <div
                    className="tlfnummer__verdi"
                  >
                    <p
                      className="typo-element"
                    >
                      Telefon 
                    </p>
                    <p
                      className="typo-normal"
                    >
                      <span>
                        +47
                         
                      </span>
                      408 53 158
                    </p>
                  </div>
                  <div
                    className="tlfnummer__knapper"
                  >
                    <button
                      className="knapp tlfnummer__knapp-med-ikon knapp--flat"
                      disabled={false}
                      onClick={[Function]}
                      type="button"
                    >
                      <div
                        className="tlfnummer__knapp-ikon"
                      >
                        <img
                          alt="Endre telefonnummer"
                          src="Pencil.svg"
                        />
                      </div>
                      <div
                        className="tlfnummer__knapp-tekst"
                      >
                        Endre
                      </div>
                    </button>
                    <button
                      className="knapp tlfnummer__knapp-med-ikon knapp--flat"
                      disabled={false}
                      onClick={[Function]}
                      type="button"
                    >
                      <div
                        className="tlfnummer__knapp-ikon"
                      >
                        <img
                          alt="Slett telefonnummer"
                          src="Slett.svg"
                        />
                      </div>
                      <div
                        className="tlfnummer__knapp-tekst"
                      >
                        Slett
                      </div>
                    </button>
                  </div>
                </div>
              </form>
            </div>
            <button
              className="tlfnummer__leggtil lenke"
              onClick={[Function]}
            >
              <span
                className="kilde__icon"
              >
                <img
                  alt="Ekstern lenke"
                  src="LeggTil.svg"
                />
              </span>
              <p
                className="typo-normal"
              >
                Legg til
              </p>
            </button>
            <div
              className="kilde__container"
            >
              <div
                className="kilde__tekst"
              >
                <p
                  className="typo-undertekst"
                >
                  Kilde: NAV
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="box__wrapper"
    id="adresser"
  >
    <div
      className="nav-veilederpanel nav-veilederpanel--kompakt nav-veilederpanel--plakat"
    >
      <div
        className="nav-veileder nav-veileder--flytende"
      >
        <div
          className="nav-veileder__frame nav-veileder__frame--m"
        >
          <img
            alt="Veileder"
            className="box__ikon"
            src="Adresse.svg"
          />
        </div>
      </div>
      <div
        className="nav-veilederpanel__content"
      >
        <div
          className="box__container"
        >
          <div
            className="box__header"
          >
            <div
              className="box__title-container"
            >
              <div
                className="box__line"
              />
              <h2
                className="typo-systemtittel box__title"
              >
                Adresser
              </h2>
              <button
                aria-label="Informasjonstekst"
                aria-pressed={false}
                className="infotekst__title-i-button"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                title="Informasjon"
              >
                <img
                  alt="Vis mer informasjon"
                  className="infotekst__title-i-icon"
                  src="Informasjonstekst.svg"
                />
              </button>
              <div
                className="box__line"
              />
            </div>
            <a
              className="lenke anchor-link"
              href="#adresser"
            >
              <img
                alt=""
                src="AnchorLink.svg"
              />
              Lenke hit
            </a>
          </div>
          <div
            className="box__content"
          >
            <div
              className="adresse__box"
            >
              
              <div>
                <div
                  className="underseksjon__header"
                >
                  <h2
                    className="typo-undertittel"
                  >
                    Adresser som er registert i Folkeregisteret
                  </h2>
                </div>
                <div
                  className="adresse__box"
                >
                  <div
                    className="underseksjon__header"
                  >
                    <p
                      className="typo-element"
                    >
                      Bostedsadresse
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      JØRGENSENVEIEN
                       
                      14
                       
                      B
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      0194
                       
                      OSLO
                    </p>
                  </div>
                  <div
                    className="lesMerPanel adresse__lesmer"
                    id="Mocked UUID"
                  >
                    <div
                      className="lesMerPanel__merContainer"
                    />
                    <div
                      className="lesMerPanel__toggle"
                    >
                      <button
                        aria-controls="Mocked UUID"
                        aria-expanded={false}
                        className="lesMerPanel__togglelink"
                        onClick={[Function]}
                        type="button"
                      >
                        <span
                          className="nav-frontend-chevron chevronboks lesMerPanel__toggleChevron chevron--ned"
                        />
                        <div
                          className="lesMerPanel__toggleTekst"
                        >
                          Vis detaljer
                        </div>
                      </button>
                    </div>
                  </div>
                  <div
                    className="adresse__divider"
                  />
                </div>
                <div
                  className="adresse__box"
                >
                  <div
                    className="underseksjon__header"
                  >
                    <p
                      className="typo-element"
                    >
                      Delt bosted
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      JØRGENSENVEIEN
                       
                      14
                       
                      B
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      0194
                       
                      OSLO
                    </p>
                  </div>
                  <div
                    className="lesMerPanel adresse__lesmer"
                    id="Mocked UUID"
                  >
                    <div
                      className="lesMerPanel__merContainer"
                    />
                    <div
                      className="lesMerPanel__toggle"
                    >
                      <button
                        aria-controls="Mocked UUID"
                        aria-expanded={false}
                        className="lesMerPanel__togglelink"
                        onClick={[Function]}
                        type="button"
                      >
                        <span
                          className="nav-frontend-chevron chevronboks lesMerPanel__toggleChevron chevron--ned"
                        />
                        <div
                          className="lesMerPanel__toggleTekst"
                        >
                          Vis detaljer
                        </div>
                      </button>
                    </div>
                  </div>
                  <div
                    className="adresse__divider"
                  />
                </div>
                <div
                  className="adresse__box"
                >
                  <div
                    className="underseksjon__header"
                  >
                    <p
                      className="typo-element"
                    >
                      Oppholdsadresse
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      JØRGENSENVEIEN
                       
                      14
                       
                      B
                    </p>
                  </div>
                  <div
                    className="adresse__linje"
                  >
                    <p
                      className="typo-normal"
                    >
                      0194
                       
                      OSLO
                    </p>
                  </div>
                  
                  <div
                    className="adresse__divider"
                  />
                </div>
                <div
                  className="kilde__container"
                >
                  <p
                    className="typo-normal"
                  >
                    <a
                      className="lenke kilde__lenke lenke"
                      href="https://www.skatteetaten.no/person/folkeregister/flytte/"
                    >
                      <span
                        className="kilde__icon"
                      >
                        <img
                          alt="Ekstern lenke"
                          src="Link.svg"
                        />
                      </span>
                      Endre adresse i Folkeregisteret
                    </a>
                  </p>
                  <div
                    className="kilde__tekst"
                  >
                    <p
                      className="typo-undertekst"
                    >
                      Kilde: Folkeregisteret
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="underseksjon__header underseksjon__divider"
              >
                <h2
                  className="typo-undertittel"
                >
                  Øvrige registrerte adresser
                </h2>
              </div>
              <div
                className="adresse__box"
              >
                <div
                  className="underseksjon__header"
                >
                  <p
                    className="typo-element"
                  >
                    Kontaktadresse registrert hos NAV
                  </p>
                </div>
                <div
                  className="adresse__linje"
                >
                  <p
                    className="typo-normal"
                  >
                    V/ Aremark Jr
                  </p>
                </div>
                
                <div
                  className="adresse__linje"
                >
                  <p
                    className="typo-normal"
                  >
                    Postboks 10 Sørstrand
                  </p>
                </div>
                <div
                  className="adresse__linje"
                >
                  <p
                    className="typo-normal"
                  >
                    0194
                     
                    OSLO
                  </p>
                </div>
                <div
                  className="lesMerPanel adresse__lesmer"
                  id="Mocked UUID"
                >
                  <div
                    className="lesMerPanel__merContainer"
                  />
                  <div
                    className="lesMerPanel__toggle"
                  >
                    <button
                      aria-controls="Mocked UUID"
                      aria-expanded={false}
                      className="lesMerPanel__togglelink"
                      onClick={[Function]}
                      type="button"
                    >
                      <span
                        className="nav-frontend-chevron chevronboks lesMerPanel__toggleChevron chevron--ned"
                      />
                      <div
                        className="lesMerPanel__toggleTekst"
                      >
                        Vis detaljer
                      </div>
                    </button>
                  </div>
                </div>
                <div
                  className="adresse__divider"
                />
              </div>
              <button
                className="kilde__lenke lenke"
                onClick={[Function]}
              >
                <span
                  className="kilde__icon"
                >
                  <img
                    alt="Ekstern lenke"
                    src="Slett.svg"
                  />
                </span>
                <p
                  className="typo-normal"
                >
                  Slett kontaktadresse
                </p>
              </button>
              <div
                className="kilde__container"
              >
                <div
                  className="kilde__tekst"
                >
                  <p
                    className="typo-undertekst"
                  >
                    Kilde: NAV
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="box__wrapper"
    id="utbetaling"
  >
    <div
      className="nav-veilederpanel nav-veilederpanel--kompakt nav-veilederpanel--plakat"
    >
      <div
        className="nav-veileder nav-veileder--flytende"
      >
        <div
          className="nav-veileder__frame nav-veileder__frame--m"
        >
          <img
            alt="Veileder"
            className="box__ikon"
            src="Kontonummer.svg"
          />
        </div>
      </div>
      <div
        className="nav-veilederpanel__content"
      >
        <div
          className="box__container"
        >
          <div
            className="box__header"
          >
            <div
              className="box__title-container"
            >
              <div
                className="box__line"
              />
              <h2
                className="typo-systemtittel box__title"
              >
                Kontonummer for utbetaling
              </h2>
              <div
                className="box__line"
              />
            </div>
            <a
              className="lenke anchor-link"
              href="#utbetaling"
            >
              <img
                alt=""
                src="AnchorLink.svg"
              />
              Lenke hit
            </a>
          </div>
          <div
            className="box__content"
          >
            
            <ul
              className="list-column-2"
            >
              <li>
                <div
                  className="listelement__header"
                >
                  <p
                    className="typo-element"
                  >
                    Kontonummer
                  </p>
                </div>
                <div
                  className="content "
                >
                  1234 56 78911
                </div>
              </li>
            </ul>
            <div
              className="kilde__container"
            >
              <button
                className="kilde__lenke lenke"
                onClick={[Function]}
              >
                <span
                  className="kilde__icon"
                >
                  <img
                    alt="Ekstern lenke"
                    src="Pencil.svg"
                  />
                </span>
                <p
                  className="typo-normal"
                >
                  Endre
                </p>
              </button>
              <div
                className="kilde__tekst"
              >
                <p
                  className="typo-undertekst"
                >
                  Kilde: NAV
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
